### PR TITLE
Add IntelliJ plugin to the list of supported tooling

### PR DESCRIPTION
**Pull request by Zelenya**

### CODE REVIEW

**Comment by Neppord** [comment]
Awesome! Much better then when it wasn't mentioned at all.

The plugin also supports formating with purs-tidy, live templates, basic refactorings like extract and inline, and also has a growing library of inspections.

**Comment by Neppord** [comment]
Oh! And treeview of tests when using spec and the teamcity reporter.

File: ecosystem/Editor-and-tool-support.md [hunk 1]
```diff
@@ -45,6 +45,10 @@ To generate `TAGS` files, use `purs docs --format etags` (or `--format ctags`).
 
 Spacemacs users can use the [PureScript layer](https://github.com/syl20bnr/spacemacs/tree/master/layers/%2Blang/purescript).
 
+### IntelliJ
+
+- [PureScript plugin](https://plugins.jetbrains.com/plugin/9738-purescript) provides editor support via `purs ide` and syntax highlighting for `.purs` files.
```
Comment (Zelenya) [inline ecosystem/Editor-and-tool-support.md:50]:
```suggestion
- [PureScript plugin](https://plugins.jetbrains.com/plugin/9738-purescript) provides editor support via `purs ide`, syntax highlighting for `.purs` files, and much more.
```

**Comment by Neppord** [comment]
Looks good to me!

**Review by sigma-andex — APPROVED** [review]

**Pull request merged**